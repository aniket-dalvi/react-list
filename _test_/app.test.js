import React from 'react';
import render from 'react-test-renderer';
import App from '../src/app';

test('app', () => {
    const app = render.create(<App/>).toJSON();
    expect(app).toMatchSnapshot();
});