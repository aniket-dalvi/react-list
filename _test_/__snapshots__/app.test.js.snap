// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app 1`] = `
<div
  className="container"
>
  <div
    className="textfield-wrapper"
  >
    <label
      className="header"
    >
      Friends List
    </label>
    <input
      onChange={[Function]}
      onKeyPress={[Function]}
      placeholder="Enter your friend's name"
      value=""
    />
  </div>
  <div
    className="no-data"
  >
    <label>
      You have not added any friend.
    </label>
  </div>
</div>
`;
